src/layman/layer/db/table.py

```diff
@@ -62,3 +63,13 @@ def get_publication_uuid(username, publication_type, publication_name):
 
 def get_metadata_comparison(username, publication_name):
     pass
+
+
+def get_layer_bbox(workspace, layer):
```

Comment:
This method is very similar to `layer.db.get_bbox`, it just returns string, whereas `get_bbox` returns list of four coordinates or None. What I prefer is to have one method that returns list of four coordinates.

I like simplicity of this method over `layer.db.get_bbox`, so I suggest to move it to current `layer.db.get_bbox` and parse string returned from DB. Also, in case of empty bbox, list of four Nones should be return instead of current single None - this requires changes in some other parts of code.

Reply:
Also, something like
```
with tmp as (
    select ST_Extent(l.wkb_geometry) as bbox
    from {workspace}.{layer} l
)
select
  st_xmin(bbox),
  st_ymin(bbox),
  st_xmax(bbox),
  st_ymax(bbox)
from tmp
```
could be used to get list of four numbers/Nones without parsing